token/cli/src/main.rs

```diff
@@ -2085,13 +2091,22 @@ fn app<'a, 'b>(
                         .takes_value(false)
                         .help("Send tokens to the recipient even if the recipient is not a wallet owned by System Program."),
                 )
+                .arg(
+                    Arg::with_name("no_recipient_is_ata_owner")
+                        .long("no-recipient-is-ata-owner")
+                        .takes_value(false)
+                        .requires("sign_only")
+                        .help("In sign-only mode, specifies that the recipient is the owner of the associated token account rather than an actual token account"),
+                )
                 .arg(
                     Arg::with_name("recipient_is_ata_owner")
                         .long("recipient-is-ata-owner")
                         .takes_value(false)
+                        .hidden(true)
```

Comment:
Please also add a `conflicts_with("no_recipient_is_ata_owner")` so only one can be specified